pkg/get/get_test.go

```diff
@@ -3508,7 +3508,7 @@ func Test_DownloadvCluster(t *testing.T) {
 func Test_DownloadHostcl(t *testing.T) {
 	tools := MakeTools()
 	name := "hostctl"
-	version := "v1.1.1"
+	version := "v1.1.3"
```

Comment:
Updated the version as ARM on Mac has seemingly only been supporter after v1.1.1